appserver/tests/payara-samples/samples/payara-expression-config-properties/src/test/java/fish/payara/samples/microprofile/config/expression/PayaraExpressionConfigPropertiesTest.java

```diff
@@ -71,14 +70,16 @@
     @Deployment(testable = false)
     public static WebArchive createDeployment() {
         return ShrinkWrap.create(WebArchive.class, "microprofile-config-expression.war")
-                .addPackage("fish.payara.samples.microprofile.config.expression")
-                .addAsManifestResource(new File("src/main/resources/META-INF/payara-expression-config.properties"));
+            .addPackage(ConfigServlet.class.getPackage()) //
```

Comment:
Why less indent? 8 spaces is the default for breaking up long lines.
Also what's with the random empty comments?

Reply:
It is called a "phantom comment". In some circumstances it is best possible way to protected readable code from automatic reformatting.
I don't see 2 block indent useful and always used 1 block, but yes, in Eclipse default is 2 blocks (8 spaces). Honestly I don't see it as a most important thing worth of discussion.